src/imgbased/command.py

```diff
@@ -32,9 +32,9 @@ def nsenter(arg, new_root=None, shell=False, environ=None):
     proc = subprocess.Popen(arg, stdout=subprocess.PIPE, env=environ,
                             stderr=subprocess.PIPE, shell=shell)
     stdout, stderr = proc.communicate()
-    log.debug("STDOUT: %s", repr(stdout))
-    log.debug("STDERR: %s", repr(stderr))
-    log.debug("ReturnCode: %s", repr(proc.returncode))
```

Comment:
It will also help to log the command. When a command fails, getting all the information about the failure helps people to report useful bugs and useful bugs help developers to resolve the issue quickly.

Reply:
> It will also help to log the command. When a command fails, getting all the information about the failure helps people to report useful bugs and useful bugs help developers to resolve the issue quickly.

We already do that, note https://github.com/lveyde/imgbased/blob/d1373b1a01e13241674d2de558c5bf7df9680b2e/src/imgbased/command.py#L31

Reply:
Currently running nsenter function (with the latest patches being applied) produces output similar to:

2022-07-07 10:22:20,076 [DEBUG] (MainThread) Executing: ['nsenter', '--root=/tmp/mnt.A7P09//', '--wd=/tmp/mnt.A7P09//', 'vdsm-tool', 'configure', '--force']
2022-07-07 10:22:41,294 [DEBUG] (MainThread) STDOUT: b"\nChecking configuration status...\n\nDB file /var/lib/vdsm/storage/managedvolume.db doesn't exists\nManaged volume database requires configuration\nsanlock is configured for vdsm\nlibvirt is not configured for vdsm yet\nSUCCESS: ssl configured to true. No conflicts\nlvm requires configuration\nDownrev multipath.conf detected, upgrade required\n\nRunning configure...\nReconfiguration of sebool is done.\nDB file /var/lib/vdsm/storage/managedvolume.db doesn't exists\nCreating managed volumes database at /var/lib/vdsm/storage/managedvolume.db\nSetting up ownership of database file to vdsm:kvm\nReconfiguration of managedvolumedb is done.\nReconfiguration of libvirt is done.\nPrevious lvmlocal.conf copied to /etc/lvm/lvmlocal.conf.20220707102238\nInstalling /usr/share/vdsm/lvmlocal.conf at /etc/lvm/lvmlocal.conf\nReconfiguration of lvm is done.\nPrevious multipath.conf copied to /etc/multipath.conf.20220707102238\nReconfiguration of multipath is done.\nReconfiguration of passwd is done.\n\nDone configuring modules to VDSM.\n"
2022-07-07 10:22:41,294 [DEBUG] (MainThread) STDERR: b''
2022-07-07 10:22:41,295 [DEBUG] (MainThread) ReturnCode: 0